scenariorun/src/main/kotlin/com/cosmotech/scenariorun/ContainerFactory.kt

```diff
@@ -1045,6 +1051,18 @@ internal class ContainerFactory(
                               "csm.platform.azure.eventBus.authentication.sharedAccessPolicy" +
                               ".namespace.name")),
               AZURE_EVENT_HUB_SHARED_ACCESS_KEY_ENV_VAR to
+                  (eventBus.authentication.sharedAccessPolicy?.namespace?.key
```

Comment:
Could you extract these (key and name, along with the null safety checks) to two reusable variables ?

Reply:
See my new review comment: https://github.com/Cosmo-Tech/cosmotech-api/pull/162#discussion_r760952775